arch/arm/src/max326xx/max32660/max32660_wdt.c

```diff
@@ -394,6 +395,8 @@ static int max326_start(struct watchdog_lowerhalf_s *lower)
 
 static int max326_stop(struct watchdog_lowerhalf_s *lower)
 {
+  struct max326_wdt_lowerhalf_s *priv =
+    (struct max326_wdt_lowerhalf_s *)lower;
```

Comment:
```suggestion
  struct max326_wdt_lowerhalf_s *priv = lower;
```

Reply:
will report warnning
chip/max32660/max32660_wdt.c:398:41: warning: initialization of 'struct max326_wdt_lowerhalf_s *' from incompatible pointer type 'struct watchdog_lowerhalf_s *' [-Wincompatible-pointer-types]
  398 |   struct max326_wdt_lowerhalf_s *priv = lower;

Reply:
Sorry, I misread the line of code, please ignore this comment